docs/source/release_notes.rst

```diff
@@ -1,6 +1,8 @@
 Release Notes
 =======================================
 
+Release notes can be found here. This is typically more comprehensive than the announcements on the GitHub Discussion board: https://github.com/orgs/MAAP-Project/discussions/categories/announcements.
```

Comment:
> Release notes can be found here.
Maybe better to explain what's in a release note?
- Major system changes
- Fixes
- Feature Removal notices
etc...